src/infrastructure/figma/transformers/figma-node-transformer.test.ts

```diff
@@ -35,41 +36,175 @@ describe('transformNodeToAtlassianDesign', () => {
 	});
 
 	it('should correctly map to atlassian design', () => {
-		const node = MOCK_CHILD_NODE;
-		const mockApiResponse = generateGetFileResponseWithNode({
-			node,
+		const fileKey = generateFigmaFileKey();
+		const node = generateChildNode({ id: '100:1' });
+		const nodeLastModified = new Date('2022-01-01T00:00:00Z');
+		const irrelevantLastModified = new Date('2023-06-15T00:00:00Z');
+		const fileResponse = generateGetFileResponse({
+			document: {
+				...MOCK_DOCUMENT,
+				children: [
+					generateFrameNode({
+						id: '1:1',
+						lastModified: irrelevantLastModified,
+						children: [
+							generateChildNode({ id: '2:1' }),
+							generateChildNode({ id: '2:2' }),
+						],
+					}),
+					generateFrameNode({
+						id: '1:2',
+						lastModified: nodeLastModified,
+						children: [generateChildNode({ id: '2:3' }), node],
+					}),
+				],
+			},
+			lastModified: irrelevantLastModified,
 		});
-		const expected: AtlassianDesign = {
-			id: `${MOCK_FILE_KEY}/${node.id}`,
+
+		const result = transformNodeToAtlassianDesign({
+			fileKey,
+			nodeId: node.id,
+			fileResponse,
+		});
+
+		expect(result).toStrictEqual({
+			id: `${fileKey}/${node.id}`,
 			displayName: node.name,
 			url: buildDesignUrl({
-				fileKey: MOCK_FILE_KEY,
-				fileName: MOCK_FILE_NAME,
+				fileKey,
+				fileName: fileResponse.name,
 				nodeId: node.id,
 			}),
 			liveEmbedUrl: buildLiveEmbedUrl({
-				fileKey: MOCK_FILE_KEY,
-				fileName: MOCK_FILE_NAME,
+				fileKey,
+				fileName: fileResponse.name,
 				nodeId: node.id,
 			}),
 			inspectUrl: buildInspectUrl({
-				fileKey: MOCK_FILE_KEY,
-				fileName: MOCK_FILE_NAME,
+				fileKey,
+				fileName: fileResponse.name,
 				nodeId: node.id,
 			}),
 			status: AtlassianDesignStatus.NONE,
 			type: AtlassianDesignType.OTHER,
-			lastUpdated: expect.anything(),
-			updateSequenceNumber: parseInt(mockApiResponse.version, 10),
-		};
+			lastUpdated: nodeLastModified.toISOString(),
+			updateSequenceNumber: nodeLastModified.getTime(),
+		});
+	});
+});
 
-		const result = transformNodeToAtlassianDesign({
-			fileKey: MOCK_FILE_KEY,
-			nodeId: node.id,
-			fileResponseWithNode: mockApiResponse,
+describe('transformNodeToAtlassianDesign', () => {
```

Comment:
Think this should be `getNodeAndNodeLastModified` instead of `transformNodeToAtlassianDesign`.

Reply:
Nice catch! My copying/pasting skill has reached the next level :D 

✅ Fixed.